Posts for June and early July governance (#647)
rust-lang/blog.rust-lang.org: posts/inside-rust/2020-06/2020-JuneJuly-governance-wg-meetings.md

author: Val Grimm
team: The Governance WG <https://github.com/rust-lang/wg-governance>
---

Hello everyone! 

This blog post summarizes the thematically-related meetings of June 4, June 25, and July 9th. 

## Big picture themes
Define the role of domain working groups and community groups in the Rust Project; Rust governance taxonomy

You can find the <br>
[detailed minutes June 4](https://github.com/rust-lang/wg-governance/blob/master/minutes/2020.06.04.md)  <br>
[detailed minutes June 25](https://github.com/rust-lang/wg-governance/blob/master/minutes/2020.06.25.md)  <br>
[detailed minutes July 9](https://github.com/rust-lang/wg-governance/blob/master/minutes/2020.07.09.md)  <br>
on the [wg-governance](https://github.com/rust-lang/wg-governance) repository, but here is a quick summary: <br>

* There is a need for more clarity about why and how groups are formed, 
    * what group categories mean, 
    * how they relate to each other, 
    * how they relate to the Rust Project
    * and their level of decision-making power
* Rust repository structure and permissions should ideally represent status of groups in the project

> * There will very likely be a proposal for Community Groups to have their own repositories outside of the Rust project and not be listed in

Reviewer: ```suggestion
* There was discussion around Community Groups having their own repositories outside of the Rust project and not be listed in
```
